app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventCardsProvider.java

```diff
@@ -0,0 +1,52 @@
+package de.tum.in.tumcampusapp.component.ui.ticket;
+
+import android.content.Context;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import androidx.annotation.NonNull;
+import de.tum.in.tumcampusapp.api.tumonline.CacheControl;
+import de.tum.in.tumcampusapp.component.ui.overview.card.Card;
+import de.tum.in.tumcampusapp.component.ui.overview.card.ProvidesCard;
+import de.tum.in.tumcampusapp.component.ui.ticket.model.Event;
+import de.tum.in.tumcampusapp.component.ui.ticket.repository.EventsLocalRepository;
+
+public class EventCardsProvider implements ProvidesCard {
+
+    private final Context context;
+
+    private final EventsLocalRepository localRepository;
+
+    @Inject
+    public EventCardsProvider(Context context, EventsLocalRepository localRepository) {
+        this.context = context;
+        this.localRepository = localRepository;
+    }
+
+    public void setDismissed(int id) {
+        localRepository.setDismissed(id);
+    }
+
+    @NotNull
+    @Override
+    public List<Card> getCards(@NonNull @NotNull CacheControl cacheControl) {
```

Comment:
I think we mostly use `androidx.annotation.NonNull` instead of `org.jetbrains.annotations.NotNull`

Reply:
Don’t know how that slipped in there. Removed now.